sdk/src/entrypoint.rs

```diff
@@ -1,3 +1,9 @@
+//! The Rust-based BPF program entry point supported by the latest BPF loader.
```

Comment:
I was going to recommend "entrypoint" be one word, but seems like we're not consistent about that. Alas, and nevermind.

Reply:
Yeah, this is just copied from the solana-program docs. I'll look at the other docs and see if we can use "entrypoint" everywhere.

Reply:
I've changed all the entrypoint docs to spell it with one word.